### PR TITLE
Fix links to docs in Readme

### PR DESCRIPTION
**Pull request by posva**
I'm unsure about the first link update because there's no real equivalent on the v2 docs. It may be worth to update the whole line

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,8 +58,8 @@ Vue.js is an MIT-licensed open source project. Its ongoing development is made p
 
 Vue.js is a library for building interactive web interfaces. It provides data-reactive components with a simple and flexible API. Core features include:
 
-- [Dead simple, unobtrusive reactivity using plain JavaScript objects.](http://vuejs.org/guide/overview.html#Reactive-Data-Binding)
-- [Component-oriented development style with tooling support](http://vuejs.org/guide/overview.html#Component-System)
+- [Dead simple, unobtrusive reactivity using plain JavaScript objects.](http://vuejs.org/guide/index.html#Declarative-Rendering)
```
Comment (yyx990803) [inline README.md]: Agreed, let's reword this to "Declarative rendering with a plain JavaScript object based reactivity system."

**Comment by posva** [comment]
@yyx990803 Done
